### PR TITLE
[mtouch] Better error handling when copying the aot files.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tools/mtouch/Application.cs [hunk 1]
```diff
@@ -1068,23 +1085,29 @@ public static void CopyMsymData (string src, string dest)
...
-					return;
+			try {
+				if (p.Start ()) {
+					var error = p.StandardError.ReadToEnd();
+					p.WaitForExit ();
+					if (p.ExitCode == 0)
+						return;
+					else {
+						Console.Error.WriteLine ($"Msym files could not be copied from {src} to {dest}: {error}.");
+						throw new MonoTouchException (95, true, "Aot files could not be copied to the destination directory: {dest}");
```
Comment (rolfbjarne) [inline tools/mtouch/Application.cs]: Throwing the warning will effectively turn it into an error.

Use `ErrorHelper.Warning` instead, which will just print it.

This also makes the `Console.Error.WriteLine` above redundant.

File: tools/mtouch/Application.cs [hunk 2]
```diff
@@ -862,6 +862,23 @@ void Initialize ()
 
 			Namespaces.Initialize ();
 
+			var hasBitcodeCapableRuntime = false;
```
Comment (rolfbjarne) [inline tools/mtouch/Application.cs]: This shouldn't be here, you're reintroducing code unrelated code (that was removed recently).

File: tools/mtouch/error.cs [hunk 3]
```diff
@@ -101,6 +101,7 @@ namespace Xamarin.Bundler {
 	//					MT0092	<used by Xamarin.Launcher> The option '{0}' is required.
 	//					MT0093	Could not find 'mlaunch'.
 	//		Warning		MT0094	<unused> Both profiling (--profiling) and incremental builds (--fastdev) are currently not supported when building for {0}, and incremental builds have been disabled (this will be fixed in a future release).
+	//					MT0095 Aot files could not be copied to the destination directory.
```
Comment (rolfbjarne) [inline tools/mtouch/error.cs]: You also need to add the new warning here: docs/website/mtouch-errors.md

File: tools/mtouch/Application.cs (hunk 2 above)
Comment (mandel-macaque) [inline tools/mtouch/Application.cs]: Strange, I did not add that, removing.

**Comment by rolfbjarne** [comment]
LGTM 👍

File: docs/website/mtouch-errors.md [hunk 4]
```diff
@@ -355,6 +355,7 @@ A potential alternative solution is to enable the managed linker (although this
 
 A last-straw solution would be to use a different version of Xamarin.iOS, one that supports the SDK your project requires.
 
+<h3><a name="MT0093"/>MT0093: Aot symbolication files could not be copied to the destination directory. Symbolication will not work with the application.</h3>
```
Comment (spouliot) [inline docs/website/mtouch-errors.md]: 👍
